src/ui/screens/settings/ViewAboutUs.qml

```diff
@@ -96,7 +96,7 @@ MZViewBase {
 
                 Layout.preferredHeight: 1
                 Layout.fillWidth: true
-                color: "#0C0C0D0A"
```

Comment:
This one is a bit odd because there's an alpha component here, which I think translates to grey10 but very faint and the alpha blending is lost by calling it "grey10"

I don't actually know if the alpha here matters at all or even makes a visible difference.